helm-chart-sources/logstream-master/README.md

```diff
@@ -4,9 +4,9 @@
 
 This Chart deploys a Cribl LogStream master server.
 
-# DISCLAIMER
+# Deprecation Notice
+As of this version, 2.9.9, this chart has been deprecated, in favor of the [logstream-leader](../logstream-leader/) chart. You *can* still run this chart with newer versions of the LogStream product (see the section on [Installing a specific version of LogStream](#versionInstall)
```

Comment:
I'd pull the comma after "deprecated" and use your credit to append a period at the end of the paragraph. 😉